src/app/pages/CreateWalletPage/index.tsx

```diff
@@ -47,6 +47,9 @@ export function CreateWalletPage(props: CreateWalletProps) {
     }
   }, [createWalletActions, dispatch])
 
+  //@TODO Remove when firefox supports backdropFilter (used inside MnemonicValidation)
+  const blurMnemonicInFirefox = showConfirmation ? { filter: 'blur(5px)' } : {}
```

Comment:
Slightly refactored and added TODO

Reply:
Thanks :+1:

Reply:
Maybe also add link to https://bugzilla.mozilla.org/show_bug.cgi?id=1578503 ?